docs/reference/templating.rst

```diff
@@ -413,22 +413,16 @@ Content Field relations
 
     {% set related_content = content.fieldRelation('related_article') %}
 
-    {% if related_content is defined %}
+    {% if related_content is not empty %}
         <a href="{{ ez_path(related_content) }}">{{ related_content.name }}</a>
-    {% else %}
```

Comment:
Why should we remove this? This provides context to the person reading the docs?

Reply:
Yes, but text needs changes. Do you have suggestions how to edit these tips? Is the first posibility is redundat now. Don't know if second posibility is valid with new condition?

Reply:
Fixed through amend.